tasks.py

```diff
@@ -686,3 +686,10 @@ def generate_self_signed(domain):
         ' -keyout {0}.key -out {0}.crt'
     ).format(domain)
     run(cmd)
+
+@task
+def update_citation_styles():
+    print("Not implemented. Do the following manually:")
```

Comment:
Sure can - the only barrier is time. I thought it better to stub the method and print the steps than to send you an email. I don't use Invoke much, so I wasn't confident I would be able to get it working in just a few minutes.